docs/receiving/source-ips.mdx

```diff
@@ -4,21 +4,33 @@ title: Source IP Addresses
 
 In case your webhook receiving endpoint is behind a firewall or NAT, you may need to allow traffic from Svix's IP addresses.
 
-This is the full list of IP addresses that webhooks may originate from:
+This is the full list of IP addresses that webhooks may originate from grouped by the region of the sender:
+
+## US
 
 ```
-54.216.8.72
-54.173.54.49
-52.215.16.239
-52.55.123.25
-52.6.93.106
-63.33.109.123
 44.228.126.217
 50.112.21.217
 52.24.126.164
 54.148.139.208
 ```
 
+## EU
+
+```
+52.215.16.239
```

Comment:
Are you sure? 4 in the US an only 3 in the EU?

Reply:
Yes! Just double-checked.